operational/azure/sync_tags_with_optima/sync_azure_tags.pt

```diff
@@ -248,7 +248,7 @@ define add_tags($data, $rs_org_id) return $all_responses do
   sub on_error: skip do
     $response = http_request(
       verb: "post",
-      host: "optima.rightscale.com",
+      host: "api.optima.flexeraeng.com",
```

Comment:
This should use the `rs_optima_host` keyword instead of further hardcoding.

Reply:
I agree, but the intent here was to get a basic fix in place ASAP since (at the time) the other endpoint had broken. Swapping a hard value for a variable across every policy that previously had it hardcoded will involve significantly more work, since it means the policies will have to be adjusted more substantially, tested, etc.